src/ontology/edits/oeo-model.omn

```diff
@@ -1047,13 +1047,42 @@ Class: OEO_00010255
         <http://purl.obolibrary.org/obo/IAO_0000118> "NDC",
         <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/947
 pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1151",
-        rdfs:label "nationally determined contribution"@de
+        rdfs:label "nationally determined contribution"
```

Comment:
Reviewers, please note: This change here is intentionally. First I added the new classes accidentally with German language labels. After discovering that, I deleted all wrong language labels in commit 11fa9a8ff84ad80125b3c168a32ec5805726bf9a.